### ISSUE TITLE
Item "Duplication" Glitch using Black Hole Units and Sophisticated Backpacks

### ISSUE THREAD

**Issue by Ryani7e** [EnigmaticaModpacks/Enigmatica6#5360]
### Modpack Version

Enigmatica 6 Expert 1.6.2

### Describe your issue.

Unsure whether this is an issue with Sophisticated Backpacks or Industrial Foregoing

1. Use a Sophisticated Backpack with any Stack Upgrade and Deposit Upgrade installed.
2. Have more than a stack of any item (64+) deposited into a Black Hole Unit. This is done by shift right-clicking the Black Hole Unit.
3. This will make a glitched Black Hole Unit that will show the items in the Unit with The One Probe, but no item picture will appear on the block. In the example below, I put 256 Granite into the Unit, which was stored in one inventory slot inside the backpack because of the Stack Upgrade. 
![image](https://user-images.githubusercontent.com/26018895/226302559-b2a6295b-11bf-476b-8200-68870618faf4.png)
No item picture for granite is visible on the Black Hole Unit.
(Right now this Black Hole Unit is glitched and will behave strangely too).
4. Break and place the Black Hole Unit down again. It will appear that there is no inventory inside it (on One Probe)
5. Right click any item into the Black Hole Unit will change the previous items into the new item. For example, right-clicking my enchanted sword into the unit will show 257 swords inside the unit, instead of granite. Left clicking the Unit will extract duplicated enchanted swords.
![image](https://user-images.githubusercontent.com/26018895/226304499-2faa6eb8-c266-427d-b510-305b5a4505fb.png)

### Latest Log

https://gist.github.com/Ryani7e/a1d8e6bbcb16ca7593391127b184730d

### Have you modified the modpack?

Yes

### User Modifications

I disabled seagulls from stealing food from me because it made me sad lol

### Did the issue happen in singleplayer or on a server?

Singleplayer

### Discord Username

Ree Anne#1942

**Comment by NielsPilgaard** [comment]
Thanks for the report. Could you please update and see if the issue persists?

**Comment by Ree-Anne** [comment]
Yes, the issue still persists in 1.8.0. Sorry for taking so long to confirm this, I was having a lot of trouble with updating
